YubiKit/YubiKit/PIV/PIVSession.swift

```diff
@@ -363,9 +571,9 @@ public final actor PIVSession: Session {
     /// - Parameters:
     ///   - managementKey: The management key as Data.
     ///   - keyType: The management key type.
-    public func authenticateWith(managementKey: Data, keyType: PIVManagementKeyType) async throws {
+    public func authenticateWith(managementKey: Data, keyType: PIV.ManagementKeyType) async throws {
```

Comment:
For our other SDKs we've decided to deprecate being able to specify `keyType` here, as we can determine it by reading the ManagementKeyMetadata. We should probably do that from the start here for Swift?